component/badge.css

```diff
@@ -0,0 +1,123 @@
+#badge-component{
+    display: flex; 
+    flex-direction: column;
+ }
+ 
+ 
+ 
+ #badge-component #badge-onicon{
+     
+ }
+ 
+ .badge{
+     display: inline-block;
+     position: relative;
+     margin: 1rem 4rem;
+ }
+ 
+ .badge .number{
+     display: flex;
+     justify-content: center;
+     align-items: center;
+     color: white;
+     height: 2rem;
+     width: 2rem;
+     border-radius: 50%;
+     position: absolute;
+     top: -0.8rem;
+     right: -1.4rem;
+ }
+ 
+ .badge-primary{
+     background-color: #0275d8;
+ }
+ 
+ .badge-secondary{
+     background-color: #576574;
+ }
+ 
+ .badge-success{
+     background-color: #5cb85c;
+ }
+ 
+ .badge-danger{
+     background-color: #d9534f;
+ }
```

Comment:
make separate css variables for success, danger, primary and secondary colors in main.css/home.css/reset.css file

Reply:
Color variable added